src/lib/rest/uriParamNames.h

```diff
@@ -45,6 +45,7 @@
 #define URI_PARAM_ATTRIBUTE_FORMAT        "attributeFormat"
 #define URI_PARAM_OPTIONS                 "options"
 #define URI_PARAM_TYPE                    "type"
+#define URI_PARAM_SORTED                  "sorted"
```

Comment:
Looking https://github.com/telefonicaid/fiware-orion/issues/1103 in detail, it tells about `sort=`.

Not sure what is the best word... `sort`, `sorted`, `sortBy` all are on the table. What do you prefer? @jmcanterafonseca feedback is also valuable.